### PR TITLE
chore(templates): sso-tab-vs

### PR DESCRIPTION
**Pull request by xzf0587**

### CODE REVIEW

File: templates/scenarios/sso-tab-csharp/appPackage/manifest.template.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+    "manifestVersion": "1.14",
+    "version": "1.0.0",
+    "id": "${{TEAMS_APP_ID}}",
+    "packageName": "com.microsoft.teams.extension",
+    "developer": {
+        "name": "Teams App, Inc.",
+        "websiteUrl": "${{TAB_ENDPOINT}}",
+        "privacyUrl": "${{TAB_ENDPOINT}}/index.html#/privacy",
```
Comment (hund030) [inline templates/scenarios/sso-tab-csharp/appPackage/manifest.template.json:10]: Blazor is hosted in webapp as a server. We don't need index.html#
Just `"${{TAB_ENDPOINT}}/privacy"`,

File: templates/scenarios/sso-tab-csharp/appPackage/manifest.template.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                "groupchat"
+            ]
+        }
+    ],
+    "staticTabs": [
+        {
+            "entityId": "index",
+            "name": "Personal Tab",
+            "contentUrl": "${{TAB_ENDPOINT}}/index.html#/tab",
+            "websiteUrl": "${{TAB_ENDPOINT}}/index.html#/tab",
```
Comment (hund030) [inline templates/scenarios/sso-tab-csharp/appPackage/manifest.template.json:43]: Same as above

File: templates/scenarios/sso-tab-csharp/teamsfx/app.local.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+version: 1.0.0
```
Comment (hund030) [inline templates/scenarios/sso-tab-csharp/teamsfx/app.local.yml:1]: Do we need this file?

Comment (xzf0587) [reply templates/scenarios/sso-tab-csharp/teamsfx/app.local.yml]: removed it.

**Review by hund030 — APPROVED** [review]

**Pull request merged**